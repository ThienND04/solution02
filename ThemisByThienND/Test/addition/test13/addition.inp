10
2934 9639 1301 642515
1392 9778 1934 560820
3175 10011 1384 706032
2219 5690 2049 659480
2199 8929 3144 686159
1940 9126 3117 719209
2532 7787 2851 669400
1213 9158 2495 587168
3034 7034 1522 744140
1326 6581 1531 612863
