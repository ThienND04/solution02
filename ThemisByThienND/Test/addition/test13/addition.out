7
11
6
9
3
6
4
7
17
15
